config/application.rb

```diff
@@ -25,6 +25,17 @@ class Application < Rails::Application
       config.paths["app/views"].unshift ::Configuration.custom_views_root.to_s
     end
 
+    # Handle case where app database file is missing and user does not use the Dashboard to launch app
+    if (! ::Configuration.database_path.file?) || ::Configuration.database_path.empty?
+      p 'Building the database'
```

Comment:
Can you use Rails logger here?

Reply:
Apparently not. Attempting to use `Rails.logger.info` reports that `Rails.logger` is `nil`.